zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/FileSystemStorage.java

```diff
@@ -149,7 +161,6 @@ public Void call() throws IOException {
 
   public synchronized <T> T callHdfsOperation(final HdfsOperation<T> func) throws IOException {
     if (isSecurityEnabled) {
-      UserGroupInformation.getLoginUser().reloginFromKeytab();
```

Comment:
why this change?

Reply:
It is not necessary to do relogin because hadoop rpc will do it.